onnxruntime/core/framework/utils.cc

```diff
@@ -765,6 +768,31 @@ common::Status ExecuteGraph(const SessionState& session_state,
 #endif
 }
 
+common::Status ExecuteGraph(const SessionState& session_state,
+                            FeedsFetchesManager& feeds_fetches_manager,
+                            gsl::span<const OrtValue> feeds, std::vector<OrtValue>& fetches,
+                            ExecutionMode execution_mode, const RunOptions& run_options,
+                            const logging::Logger& logger) {
+#ifdef USE_CLOUD
+  if (run_options.config_options.configurations.count("use_cloud") &&
```

Comment:
Prefer using find and the returned iterator.

Reply:
Done.